source/tutorials/smb-server.rst

```diff
@@ -0,0 +1,149 @@
+.. _smb-server:
+
+Samba Server
+############
+
+This tutorial describes how to enable simple file sharing on a system
+running |CL-ATTR| and how to access the share from clients on other operating 
+systems. 
+
+.. contents::
+   :local:
+   :depth: 1
+
+Prerequisites
+*************
+
+* |CL| is installed on your host system.
+
+  For detailed instructions, follow one of these guides:
+
+   * *Desktop* version: :ref:`bare-metal-install-desktop`
+   * *Server* version: :ref:`bare-metal-install-server`
```

Comment:
```suggestion
  * *Server* version: :ref:`bare-metal-install-server`
```
Should line up with the F in For to render properly.

Reply:
fixed.